source/more-info/unhealthy/supervisor.markdown

```diff
@@ -15,3 +15,5 @@ This can also be done with the CLI, by running the following command:
 ```bash
 ha supervisor update
 ```
+
+In case this doesn't work. Check the logs, if there's an error where a host can't be resolved (eg: lookup ghcr.io: no such host) there might be a problem with your dns settings.
```

Comment:
Valid remark. Thought, the reason I was able to resolve this was because I found a forum post that had the exact same issue. Which lead me to realize that I had taken my dns server offline and reverted to my ISP's dns. Resulting in me realising that it might be set to a static value, resolving my issue. Since one of the first community posts I found had the exact same issue as me, I personally feel it's important to include the dns thing. Probably a lot of people running homeassistant also set up a pi-hole.